QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -15,21 +18,25 @@
     public static final BooleanStatus raising = new BooleanStatus();
     public static final BooleanStatus lowering = new BooleanStatus();
 
-    public static final BooleanInputPoll topLimitSwitch = Igneous.makeDigitalInput(0);
-    public static final BooleanInputPoll bottomLimitSwitch = Igneous.makeDigitalInput(1);
+    public static final BooleanInput topLimitSwitch = BooleanMixing.createDispatch(Igneous.makeDigitalInput(0), Igneous.globalPeriodic);
+    public static final BooleanInput bottomLimitSwitch = BooleanMixing.createDispatch(Igneous.makeDigitalInput(1), Igneous.globalPeriodic);
 
     public static EventInput raisingInput;
     public static EventInput loweringInput;
 
     public static void setup() {
-        raising.setFalseWhen(EventMixing.filterEvent(topLimitSwitch, true, Igneous.globalPeriodic));
-        lowering.setFalseWhen(EventMixing.filterEvent(bottomLimitSwitch, true, Igneous.globalPeriodic));
+        raising.setFalseWhen(BooleanMixing.onPress(topLimitSwitch));
+        lowering.setFalseWhen(BooleanMixing.onPress(bottomLimitSwitch));
```

Comment:
No. This means that it only stops when they're pressed - so then there's the bug of pressing up when already at the top.